lib/builder-webpack5/templates/virtualModuleModernEntry.js.handlebars

```diff
@@ -32,7 +34,7 @@ window.__STORYBOOK_CLIENT_API__ = new ClientApi({ storyStore: preview.storyStore
 
 preview.initialize({ importFn, getProjectAnnotations });
 
-if (module.hot) {
+if (import.meta.webpackHot) {
```

Comment:
Is this a backwards compatible change?

Reply:
The folder structure indicates, that this template is only used for the webpack 5 builder. As I have understood the docs, it is totally valid to use this syntax also for CommonJS environments (https://webpack.js.org/api/hot-module-replacement/). So if your question is about backwards-compatibility regarding CommonJS environments, then yes. It should work. Especially, because it is already used in the next line.